instrumentation/kamon-instrumentation-common/src/main/scala/kamon/instrumentation/http/HttpInstrumentation.scala

```diff
@@ -0,0 +1,21 @@
+package kamon.instrumentation.http
+
+import kamon.util.Filter
+
+object HttpInstrumentation {
+
+  final case class OperationNameSettings(defaultOperationName: String,
+                                         operationMappings: Map[Filter.Glob, String],
+                                         operationNameGenerator: HttpOperationNameGenerator)
+
+  private[http] def operationName(settings: OperationNameSettings, request: HttpMessage.Request): String = {
```

Comment:
Why not make this method a member of the `OperationNameSettings` class? :thinking:

Reply:
yeah don't see why not! i've made that change